packages/chatbot-server-mongodb-public/src/evalEnvVars.ts

```diff
@@ -0,0 +1,6 @@
+export const evalEnvVars = {
```

Comment:
```suggestion
export const EVAL_ENV_VARS = {
```

For consistency - we're using SCREAMING_SNAKE_CASE for these in other places.

Also super nitty, the file name should use PascalCase instead of camelCase